### DISCUSSION TITLE
Auto Updater/In Game Updates

### DISCUSSION THREAD

**Discussion by tedztar** [Zrips/CMI#2303] [Feature Requests]
**Description of feature/improvement request:**
The ability to update the plugin in-game (Auto-update would be nice but don't think it is possible with spigot)
---
### Detailed explanation of what it should do, when and in what situation: ###
The ability to update in-game would be really nice, if you have an update available you could do /cmi update to bring up a menu where you would need to log in to spigot (for verification you own cmi) and it would auto-update then either restart at a set time, or reloads the plugin on the new version without restarting.

**Comment by TheBeast2155** [comment]
+1
